components/home/list/LaunchesList.jsx

```diff
@@ -0,0 +1,36 @@
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    FlatList,
+  } from "react-native";
+  import { useRouter } from "expo-router";
+  
+  import styles from "./Landing.style";
+  
+  
+  function LaunchesList(list) {
+    const router = useRouter();
+    return (
+        <View style={styles.container}>
+        {/* TODO: Is flat list more appropriate here than mapping */}
+        <FlatList
+          data={list}
+          renderItem={({ item }) => (
```

Comment:
we have a useful util in the mobile-app repo called the generateComponent util that makes this kind of renderItem very easy to implement and test. I'd recommend implementing something like that here